src/Core/src/Platform/iOS/MauiRefreshView.cs

```diff
@@ -166,17 +168,6 @@ bool TryInsertRefresh(UIView view, int index = 0)
 			return false;
 		}
 
-		public override CGRect Bounds
-		{
-			get => base.Bounds;
-			set
-			{
-				base.Bounds = value;
-				if (_contentView != null)
-					_contentView.Frame = value;
```

Comment:
This was most likely here to workaround the fact that we didn't have the layout path setup correctly. 

It looks like the original addition of this didn't add tests
https://github.com/dotnet/maui/pull/14302

So, it'd be good to test through those closed issues

Reply:
I ran the sample [here](https://github.com/dotnet/maui/issues/11327) and it looks good still

![image](https://github.com/dotnet/maui/assets/5375137/66081519-6403-4828-8e3c-ea252a88d181)